pkg/populate/populate.go

```diff
@@ -33,14 +34,16 @@ func PopulateTestFromFinished(test *v1.TestResult, test_finished artifacts.Artif
 }
 
 func PopulateTestFromStepFinished(test *v1.TestResult, step_test_finished artifacts.ArtifactResult) error {
-	test.StepExecuted = true
+	test.StepExecuted = false
 	if step_test_finished.Json["passed"] != nil {
 		test.StepPassed = step_test_finished.Json["passed"].(bool)
+		test.StepExecuted = true
```

Comment:
A more senior go programmer than me told me off for this.  See: https://stackoverflow.com/questions/31581867/is-it-required-to-assign-default-value-to-variable

As a C programmer, however, I completely understand you are doing this.

Reply:
an old (2004) Python mantra [says](https://www.python.org/dev/peps/pep-0020/):
> Explicit is better than implicit.

;-)

thanks, I'll fix it

Reply:
> an old (2004) Python mantra [says](https://www.python.org/dev/peps/pep-0020/):
> 
> > Explicit is better than implicit.

I know and I agree.  The conflicting world of programming languages...